README.md

```diff
@@ -785,6 +785,53 @@ func NewHTTPServer(host, port, applicationEnv, applicationName string) *HTTPServ
 }
 ```
 
+### gRPC server and client interceptors
+
+`go-sdk` ships with gRPC instrumentation, based on DataDog's
+`dd-trace-go` library. It creates new gRPC interceptors for the server & client thus
+requests will create traces that will be sent opaquely to the DataDog agent.
```

Comment:
What do you think about changing it to: `It creates new gRPC interceptors for the server & client. Thus, requests ...`. It looks a bit more clear.